{"name": "Проверка получения заказа конкретного неавторизованного пользователя", "status": "passed", "description": "Нельзя получить заказ конкретного неавторизованного пользователя", "start": 1723215305949, "stop": 1723215306183, "uuid": "f08cd224-30ff-4893-99ae-b70688f86e5b", "historyId": "41276ae343a457b1cdc51768f975f3cb", "testCaseId": "41276ae343a457b1cdc51768f975f3cb", "fullName": "tests.test_order.TestReceivingUserOrders#test_receiving_user_orders_not_auth_user_error_message", "labels": [{"name": "epic", "value": "test_receiving_user_orders"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order"}, {"name": "subSuite", "value": "TestReceivingUserOrders"}, {"name": "host", "value": "DESKTOP-619AU3M"}, {"name": "thread", "value": "12804-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order"}]}